### DISCUSSION TITLE
Question: why 2 test cases generated 3 json files

### DISCUSSION THREAD

**Discussion by stevez** [anishkny/playwright-test-coverage#99] [Q&A]
I just found a strange that thing in my code base: usually playwright coverage JSON files are created per test case, but I have 2 test cases but 3 files are created. I wonder in what case this will happen and if it will have any potential issues?

**Comment by kachkaev** [comment]
Can it be 1 from server and 2 from browser?

**Comment by stevez** [comment]
I found an interesting thing: if I run tests in sequential mode, then I got 2 files for 2 tests, if I run in parallel mode, then 3 files are created, this is the first time I met, seems it is related to the application

**Comment by anishkny** [comment]
This is interesting. Although it sounds like its not causing unexpected behavior?
